[2024-11-28T11:13:21.267+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: challenge_dag.load_task manual__2024-11-28T11:13:14.192164+00:00 [queued]>
[2024-11-28T11:13:21.272+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: challenge_dag.load_task manual__2024-11-28T11:13:14.192164+00:00 [queued]>
[2024-11-28T11:13:21.272+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-11-28T11:13:21.282+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_task> on 2024-11-28 11:13:14.192164+00:00
[2024-11-28T11:13:21.286+0000] {standard_task_runner.py:57} INFO - Started process 49266 to run task
[2024-11-28T11:13:21.288+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'challenge_dag', 'load_task', 'manual__2024-11-28T11:13:14.192164+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/challenge_dag.py', '--cfg-path', '/tmp/tmpktluct5b']
[2024-11-28T11:13:21.289+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask load_task
[2024-11-28T11:13:21.313+0000] {task_command.py:410} INFO - Running <TaskInstance: challenge_dag.load_task manual__2024-11-28T11:13:14.192164+00:00 [running]> on host codespaces-c5d7e2
[2024-11-28T11:13:21.350+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='challenge_dag' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T11:13:14.192164+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-28T11:13:14.192164+00:00'
[2024-11-28T11:13:21.351+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-28T11:13:21.351+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo -e ".separator ","\n.import --skip 1 /workspaces/hands-on-introduction-data-engineering-4395021/lab/challenge/challenge-transform-data.csv sp_500_sector_count" | sqlite3 /workspaces/hands-on-introduction-data-engineering-4395021/lab/challenge/challenge-load-db.db']
[2024-11-28T11:13:21.359+0000] {subprocess.py:86} INFO - Output:
[2024-11-28T11:13:21.365+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-28T11:13:21.378+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=challenge_dag, task_id=load_task, execution_date=20241128T111314, start_date=20241128T111321, end_date=20241128T111321
[2024-11-28T11:13:21.420+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-11-28T11:13:21.428+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
